### PR TITLE
add azure release 17.1.0

### PR DESCRIPTION
**Pull request by whites11**
@T-Kukawka you can use this as base for 17.1.0.
You only need to bump azure-operator to whatever version Nikola will come up with.
We also need to ensure azure-operator has k8scloudconfig >= 13.1.0

### CODE REVIEW

**Comment by uvegla** [comment]
I basically need to bump `app-operator` to `v5.12.0` so that is perfect! (Please do not use `v6.0.0` yet until https://github.com/giantswarm/giantswarm/issues/22360 is fixed.)

The other one is need is `chart-operator` to `v2.21.1` at least so `v2.22.0` that you have now works for me. There are 2 more minor version tho in `chart-operator` tho that I would recommend bumping - so [v2.24.0](https://github.com/giantswarm/chart-operator/releases/tag/v2.24.0) at the moment - if it is not too much trouble.

**Comment by whites11** [comment]
ok no problem @uvegla, please go on and bump chart operator in this PR.
how about cluster-operator? Is it mandatory to bump it now as well? We need some more work before we can do that

**Comment by uvegla** [comment]
Well ,y understanding that cluster-operator for azure is special as there is some reason for using a 3.x version still (don't know the details). Reason I wanted to bump is because [v4.3.0](https://github.com/giantswarm/cluster-operator/releases/tag/v4.3.0) has a fix for handling App Bundles (e.g. security-pack).

My main concerns are app and chart-operator tho. Gimme a sec to bump chart-operator in this PR.

File: azure/v17.1.0/README.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
+# :zap: Giant Swarm Release v17.1.0 for Azure :zap:
+
+This is a maintenace azure release that provides the latest Kubernetes 1.22 version as well as the latest version of all giantswarm releases.
```
Comment (njuettner) [inline azure/v17.1.0/README.md:3]:
```suggestion
This is a maintenance azure release that provides the latest Kubernetes 1.22 version as well as the latest version of all giantswarm releases.
```

**Review by T-Kukawka — APPROVED** [review]
Fixed 1 typo in url in announcement, LGTM

**Review by njuettner — APPROVED** [review]

**Review by alex-dabija — APPROVED** [review]

**Pull request merged**